runtime/src/main.rs

```diff
@@ -28,14 +28,7 @@ use core::hint::black_box;
 #[cfg(feature = "std")]
 pub fn main() {}
 
-const BANNER: &str = r#"
```

Comment:
Was this just to save space in 1.x?

I'll miss it. :) We aren't as space-constrained in 2.x so we could maybe keep it.

Reply:
Ya, just space savings. I'm inclined to leave it off since I think we'll end up space constrained on 2.x as well.